### DISCUSSION TITLE
Adding integration for shadowserver reports

### DISCUSSION THREAD

**Discussion by security-companion** [Security-Onion-Solutions/securityonion#13934] [Ideas] [answered]
Hi,
shadowserver created a Elasticsearch Custom Logs integration for ingesting their reports.
https://github.com/The-Shadowserver-Foundation/api_utils/tree/main/elasticsearch#custom-logs-integration

Would it be possible to add this integration to security-onion?
Best regards
security-companion

**Comment by cm-ops** [comment, marked as answer]
This is not an Elastic integration https://www.elastic.co/integrations/data-integrations?search=, it looks like you can use a custom integration configuration to add this if you wanted.

```
Custom Logs integration
Select Change defaults and Advanced options to fine tune the integration.

The drop_fields settings from the filebeat example can be entered in the Processors area.

The top level json settings from the filebeat example can be entered in the Custom configurations area.
```
